.github/workflows/on_push.yml

```diff
@@ -143,11 +143,52 @@ jobs:
         name: "Integration tests"
         run: make test-integration
 
+  version:
+    name: Version
+    concurrency: tagging
+    if: ${{ github.ref == 'refs/heads/release' || github.ref == 'refs/heads/develop' }}
+    runs-on: ubuntu-latest
+    needs: [
```

Comment:
I like it all on the same line, since we can't use trailing commas here

Reply:
My intention here was to make it more readable since the line was too long.

Reply:
Is this a YAML list? If so, we can remove `[,]` and just put each dependency on it's own line without commas.

Reply:
From [the docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds) it's not clear if it must be a string or can be a YAML list.